packages/gatsby-admin/src/components/recipes-gui/resource-message.js

```diff
@@ -0,0 +1,68 @@
+/** @jsx jsx */
+/* eslint-disable */
+import React from "react"
+import { MdRefresh, MdBrightness1 } from "react-icons/md"
+import { keyframes } from "@emotion/core"
+import { BaseAnchor, SuccessIcon } from "gatsby-interface"
+import { makeResourceId } from "./utils"
+import { jsx } from "theme-ui"
+
+const ResourceMessage = ({ state, resource }) => {
+  let icon = (
+    <MdBrightness1
+      sx={{ height: `10px`, width: `15px`, display: `inline-block` }}
+    />
+  )
+  let message = resource.describe
+
+  if (state.value === `applyingPlan` && resource.isDone) {
+    icon = <SuccessIcon />
+  } else if (state.value === `applyingPlan`) {
+    const keyframe = keyframes`
```

Comment:
Nitpick: I don't know how emotion works, but I do know that in styled-components we recommend moving `keyframes` declarations out of the render method. There is no need for this to be re-created every render! 👍